MinVer/MinVer.targets

```
@@ -7,14 +7,15 @@
   </PropertyGroup>
 
   <Target Name="MinVer_GetVersion" Condition=" '$(MinVerVersion)' == '' ">
+    <Exec Command="dotnet &quot;$(MSBuildThisFileDirectory)MinVer.Cli.dll&quot; &quot;$(MSBuildProjectDirectory)&quot;" ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" ItemName="MinVerConsoleOutputItems" />
+    </Exec>
+    <ItemGroup>
+      <MinVerConsoleOutputVersion Include="@(MinVerConsoleOutputItems)" Condition="'$([System.String]::new(%(Identity)).StartsWith(`MinVer:`))' != 'true'" />

```

Oooh! Nice idea! 💯

---

This needs a tweak though. I hacked the `test-package` target to actually make a commit before building (I'll get the change into `master` ASAP), and I got:

> MinVer.targets(14,73): error MSB4184: The expression "[System.String]::new(MinVer: Using commit    { Sha: 438d6061081fbb103005498cab6096c6b88679ab, Height: 0, Tag: '(none)', Version: 0.0.0-alpha.0 })" cannot be evaluated.

I guess it needs to be wrapped in quotes?

```suggestion
      <MinVerConsoleOutputVersion Include="@(MinVerConsoleOutputItems)" Condition="'$([System.String]::new(&quot;%(Identity)&quot;).StartsWith(`MinVer:`))' != 'true'" />
```